deployment/run.sh

```diff
@@ -47,6 +47,13 @@ run_groovy_from_source() {
         --language groovy --cast-numbers
 }
 
+run_typescript() {
+    npm install -g typescript@next
```

Comment:
Typescript should be installed in the setup script. If you want to reinstall the nightly build, then wrap that functionality in an install_nightly function.